source/api.rst

```diff
@@ -0,0 +1,71 @@
+.. toctree::
+
+API
+---
+
+It's possible to do all operations using the Onedata API.
```

Comment:
```suggestion
All operations can be performed using the Onedata API
```

Reply:
Missing ending period, could you add it?

Reply:
Implemented